lib/engine/ability/tile_lay.rb

```diff
@@ -7,7 +7,7 @@ module Ability
     class TileLay < Base
       attr_reader :hexes, :tiles, :free, :discount, :special, :connect, :blocks, :reachable
 
-      def setup(hexes:, tiles:, free: false, discount: nil, special: nil,
+      def setup(tiles:, hexes: nil, free: false, discount: nil, special: nil,
```

Comment:
How is this change tied to the PR?

Reply:
Its not. I tried to prune these changes from all the 1828 company ability changes I'm making. I saw this made it in, but didn't remove because it is an innocuous change and will be built upon in my next PR.